docs/content/flags.md

```diff
@@ -26,10 +26,10 @@ These flags are available for every command.
   -c, --checksum                             Skip based on checksum (if available) & size, not mod-time & size
       --client-cert string                   Client SSL certificate (PEM) for mutual TLS auth
       --client-key string                    Client SSL private key (PEM) for mutual TLS auth
-      --compare-dest string                  use DIR to server side copy flies from.
```

Comment:
Unfortunately this is a generated file...

The correct file which needs patching is

https://github.com/rclone/rclone/blob/53d55ae7608cdd1e07827aaa9dbaecf32764126a/fs/config/configflags/configflags.go#L72-L73

Can you patch that instead please?

Reply:
yes